hibernate-core/src/main/java/org/hibernate/boot/jaxb/spi/Binder.java

```diff
@@ -31,5 +31,14 @@ public interface Binder<T> {
 	 * @param origin The descriptor of the stream origin
 	 * @return The bound JAXB model
 	 */
-	<X extends T> Binding<X> bind(InputStream stream, Origin origin);
+//	<X extends T> Binding<X> bind(InputStream stream, Origin origin);
```

Comment:
Just to make sure its on the todo... remove this commented-out method and javadoc